src/api/app/controllers/webui/user_controller.rb

```diff
@@ -38,6 +38,7 @@ def save
 
     if @configuration.accounts_editable?(@displayed_user)
       @displayed_user.assign_attributes(params[:user].slice(:realname, :email, :in_beta).permit!)
+      RabbitmqBus.send_to_bus('metrics', "user.beta changed_to=#{@displayed_user.in_beta}") if @displayed_user.in_beta_changed?
```

Comment:
Shouldn't it be `"user.in_beta changed_to...`

Reply:
It's just a label, but you are right it would make sense to align this with the attribute name:+1: